server/src/main/java/org/candlepin/auth/KeycloakOIDCFacade.java

```diff
@@ -116,8 +114,8 @@ public InputStream getInputStream() {
 
         @Override
         public InputStream getInputStream(boolean buffered) {
-            if (inputStream != null) {
-                return inputStream;
+            if (this.inputStream != null) {
+                return this.inputStream;
```

Comment:
Good, but this still is never used, so it serves no purpose. The input stream will always be fetched from the requestContext.